stm/stm.ml

```diff
@@ -1928,86 +1930,69 @@ end (* }}} *)
 
 and Partac : sig
 
-  val vernac_interp :
-    solve:bool -> abstract:bool -> cancel_switch:AsyncTaskQueue.cancel_switch ->
-    int -> CoqworkmgrApi.priority -> Stateid.t -> Stateid.t -> aast -> unit
+  val enable_par : unit -> unit
```

Comment:
This makes Partac not recursive with the other modules (they don't call enable_par) so it should be separated from the recursive block AFAICT

Reply:
ok, I'll do it